query-node/mappings/src/content/channel.ts

```diff
@@ -77,13 +85,37 @@ export async function content_ChannelCreated(ctx: EventContext & StoreContext):
       inconsistentState(`storageBag for channel ${channelId} does not exist`)
     }
 
-    const metadata = deserializeMetadata(ChannelMetadata, channelCreationParameters.meta.unwrap()) || {}
-    await processChannelMetadata(ctx, channel, metadata, dataObjects)
+    const appAction = deserializeMetadata(AppAction, channelCreationParameters.meta.unwrap())
+
+    if (appAction && Object.keys(appAction).length) {
```

Comment:
This is now equivalent to:
```suggestion
    if (appAction) {
```